generators/common/files.js

```diff
@@ -54,6 +54,17 @@ const commonFiles = {
                 }
             ]
         }
+    ],
+    documentation: [
+        {
+            path: 'src/main/asciidoc/',
+            templates: ['index.adoc', 'auth.adoc', 'db.adoc']
```

Comment:
'db.adoc' should be added conditionally: a microservice can have no database.

Reply:
Yes indeed